rxsample/src/androidTest/java/com/mateuszkoslacz/moviper/rxsample/utils/di/TestRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.mateuszkoslacz.moviper.rxsample.utils.di;
+
+
+import com.mateuszkoslacz.moviper.rxsample.data.rdp.repository.Repository;
+
+public interface TestRepository<T> extends Repository<T> {
```

Comment:
Why did you commit this file? I asked you to commit only the first part of these tests – injecting fake repo.